### PR TITLE
Update env.ac

### PR DESCRIPTION
**Pull request by FlamePrince**
thank you [@pappons](https://github.com/pappons) for fix.

## Changes Proposed:
-  modified \conf \dist\env.ac with " around db values, syncing with the values displayed elsewhere.

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/8637

## Tests Performed:

- attempted to follow https://www.azerothcore.org/wiki/install-with-docker w/o changes, reported similar issues as found in https://github.com/azerothcore/azerothcore-wotlk/issues/8637. once I changed the values, i was able to proceed.

## How to Test the Changes:
Git GUI/bash https://github.com/azerothcore/azerothcore-wotlk.git into a new folder. Attempt to use docker-compose to follow https://www.azerothcore.org/wiki/install-with-docker. Encounter https://github.com/azerothcore/azerothcore-wotlk/issues/8637.

Amend your \conf\dist\env.ac, changing DB_AUTH_CONF=MYSQL_USER='root'; MYSQL_PASS='password'; MYSQL_HOST='ac-database'; MYSQL_PORT='3306'; to DB_AUTH_CONF="MYSQL_USER='root'; MYSQL_PASS='password'; MYSQL_HOST='ac-database'; MYSQL_PORT='3306';" 

repeat the above for DB_CHARACTERS and DB_WORLD_CONF.

## Known Issues and TODO List:

- change env.ac to allow for docker testers to more easily setup ac servers following https://www.azerothcore.org/wiki/install-with-docker 
- **or** implement [this](https://www.chromiecraft.com/how-to-install-a-wow-server-on-your-own-computer/) as the standard process, as it worked for me w/ no issues.

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: conf/dist/env.ac [hunk 1]
```diff
@@ -19,11 +19,11 @@ AC_CCACHE=true
 
 OUTPUT_FOLDER=/azerothcore/var/build/sql/
 
-DB_AUTH_CONF=MYSQL_USER='root'; MYSQL_PASS='password'; MYSQL_HOST='ac-database'; MYSQL_PORT='3306';
+DB_AUTH_CONF=MYSQL_USER="'root'; MYSQL_PASS='password'; MYSQL_HOST='ac-database'; MYSQL_PORT='3306';"
```
Comment (temperrr) [inline conf/dist/env.ac:22]: " on wrong plaace, should be "MYSQL_USER='root'; MYSQL_PASS='password'; MYSQL_HOST='ac-database'; MYSQL_PORT='3306';"

**Review by temperrr — CHANGES_REQUESTED** [review]

File: conf/dist/env.ac (hunk 1 above)
Comment (FlamePrince) [reply conf/dist/env.ac]: fixed!

**Review by temperrr — APPROVED** [review]

**Comment by temperrr** [comment]
Tested, works as expected on Windows Docker.

**Comment by Nyeriah** [comment]
Hey, please don't update your branch. I've manually started the docker CI runner and updating it seems to cancel it. I'll update your branch again once the merge time comes.

**Comment by FlamePrince** [comment]
> Hey, please don't update your branch. I've manually started the docker CI runner and updating it seems to cancel it. I'll update your branch again once the merge time comes.

oh, my bad! sorry about that

**Pull request merged**